src/evaluations/tests/analyzer_test.py

```diff
@@ -163,13 +163,13 @@ def test_get_relative_error_stats_of_num_of_estimable_sets(self):
         [analyzer.ERROR_MARGIN_NAME, analyzer.PROPORTION_OF_RUNS_NAME,
          analyzer.SKETCH_ESTIMATOR_NAME, analyzer.SCENARIO_NAME,
          analyzer.NUM_ESTIMABLE_SETS,
-         simulator.RELATIVE_ERROR + '_mean',
-         simulator.RELATIVE_ERROR + '_std'],
+         simulator.RELATIVE_ERROR_BASENAME + '1_mean',
```

Comment:
Given you use "1_mean" twice, it might be nice to store in a variable. For future proofing sake.